macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -45,12 +46,19 @@ object ContextWrapper {
 
   def apply[F](fragment: F)(implicit fragmentImpl: Fragment[F]): ActivityContextWrapper =
     ContextWrapper(fragmentImpl.activity(fragment))
+
+  def apply(view: View): ContextWrapper = view.getContext match {
+    case activity: Activity => ContextWrapper(activity)
+    case service: Service => ContextWrapper(service)
+    case app: Application => ContextWrapper(app)

```

Comment:
See the last part of this https://github.com/47deg/macroid/issues/73#issuecomment-178764093

Additionally, if you *happen* to be doing interop with xml layout resources and appcompat, if you use `android:theme` you will also encounter `ContextThemeWrapper`